pkg/storage/utils/decomposedfs/upload/upload.go

```diff
@@ -278,14 +278,10 @@ func (session *OcisSession) ConcatUploads(_ context.Context, uploads []tusd.Uplo
 func (session *OcisSession) Finalize() (err error) {
 	ctx, span := tracer.Start(session.Context(context.Background()), "Finalize")
 	defer span.End()
-	n, err := session.Node(ctx)
-	if err != nil {
-		return err
-	}
 
 	// upload the data to the blobstore
 	_, subspan := tracer.Start(ctx, "WriteBlob")
-	err = session.store.tp.WriteBlob(n, session.binPath())
+	err = session.store.tp.WriteBlob(session.SpaceID(), session.ID(), session.Size(), session.binPath())
```

Comment:
@aduffeck this is the actual change ... but I need to check what happens after this ... or betdter yet: write a test ;-)

Reply:
added an async test 💪

Reply:
This change only affects the positive case when an upload has to be transferred to the blobstore. This has no effect on failure cases.